README.rst

```diff
@@ -1,192 +1,79 @@
 .. code-block::
 
-  # Activate the virtualenv
-  workon openedx_course_export
-
-  # Grab the latest code
-  git checkout main
-  git pull
-
-  # Install/update the dev requirements
-  make requirements
-
-  # Run the tests and quality checks (to verify the status before you make any changes)
-  make validate
-
-  # Make a new branch for your changes
-  git checkout -b <your_github_username>/<short_description>
-
-  # Using your favorite editor, edit the code to make your change.
-  vim ...
-
-  # Run your new tests
-  pytest ./path/to/new/tests
-
-  # Run all the tests and quality checks
-  make validate
-
-  # Commit all your changes
-  git commit ...
-  git push
-
-  # Open a PR and ask for review.
+    # If running devstack in docker, first open a shell in CMS (make studio-shell)
 
-Deploying
-=========
+    pip install -e git+https://github.com/edly-io/openedx_course_export.git@0.1.0#egg=openedx_course_export
 
-TODO: How can a new user go about deploying this component? Is it just a few
-commands? Is there a larger how-to that should be linked here?
 
-PLACEHOLDER: For details on how to deploy this component, see the `deployment how-to`_
+Configuration
+------------
 
-.. _deployment how-to: https://docs.openedx.org/projects/openedx_course_export/how-tos/how-to-deploy-this-component.html
+**1) edx-platform configuration**
 
-Getting Help
-************
+You might need to add the following configuration values to the config file in Open edX. For any release after Juniper, that config file is ``/edx/etc/studio.yml``. These should be added to the top level. **Ask a fellow developer or devops for these values.**
 
-Documentation
-=============
-
-PLACEHOLDER: Start by going through `the documentation`_.  If you need more help see below.
-
-.. _the documentation: https://docs.openedx.org/projects/openedx_course_export
-
-(TODO: `Set up documentation <https://openedx.atlassian.net/wiki/spaces/DOC/pages/21627535/Publish+Documentation+on+Read+the+Docs>`_)
-
-More Help
-=========
-
-If you're having trouble, we have discussion forums at
-https://discuss.openedx.org where you can connect with others in the
-community.
-
-Our real-time conversations are on Slack. You can request a `Slack
-invitation`_, then join our `community Slack workspace`_.
-
-For anything non-trivial, the best path is to open an issue in this
-repository with as many details about the issue you are facing as you
-can provide.
-
-https://github.com/openedx/openedx_course_export/issues
-
-For more information about these options, see the `Getting Help`_ page.
-
-.. _Slack invitation: https://openedx.org/slack
-.. _community Slack workspace: https://openedx.slack.com/
-.. _Getting Help: https://openedx.org/getting-help
-
-License
-*******
-
-The code in this repository is licensed under the AGPL 3.0 unless
-otherwise noted.
+.. code-block::
 
-Please see `LICENSE.txt <LICENSE.txt>`_ for details.
 
-Contributing
-************
+    AWS_ACCESS_KEY_ID: <your aws access id>
```

Comment:
@Ali-Salman29 we don't need to mention this in readme. Mostly on prod IMProfile is used to manage s3 access

Reply:
Can we change the readme suggesting adding these keys to studio.yml for testing purposes?